jobs/e2e_node/containerd/image-config-systemd.yaml

```diff
@@ -0,0 +1,9 @@
+images:
+  ubuntu:
+    image_family: ubuntu-2204-lts
```

Comment:
We should apply same change as https://github.com/kubernetes/test-infra/pull/28190 here